docs/source/ros_api.rst

```diff
@@ -160,6 +160,22 @@ Once it is done processing all data, it writes out the final Cartographer state
 .. _offline_node: https://github.com/googlecartographer/cartographer_ros/blob/master/cartographer_ros/cartographer_ros/offline_node_main.cc
 
 
+Published Topics
+----------------
+
```

Comment:
Maybe add "In addition to the topics that are also published by the online node, the offline node publishes also:"